lib/hammer_cli_katello/repository.rb

```diff
@@ -219,7 +219,7 @@ class UpdateCommand < HammerCLIKatello::UpdateCommand
       option "--publish-via-http", "ENABLE", _("Publish Via HTTP"),
              :attribute_name => :option_unprotected,
              :format => HammerCLI::Options::Normalizers::Bool.new
-      option "--docker-tag", "TAG", _("Container Image tag")
+      option "--docker-tag", "TAG", _("Optional custom Container Image tag to specify the value of the Docker digest")
```

Comment:
Rubocop fails because the line is too long.

```suggestion
      option "--docker-tag", "TAG",
        _("Optional custom Container Image tag to specify the value of the Docker digest")
```

Reply:
Ah! Will fix.